src/routes/productRoutes.js

```diff
@@ -10,5 +10,8 @@ router.get('/', ProductController.findAllproducts);
 
 // CREATE SINGLE PRODUCT
 router.post('/', isSeller, ProductController.addProduct);
+// router.get('/allProduct', isActive, productController.findAllproducts);
+// router.post('/addProduct', isActive, productController.addProduct);
+router.post('/all', ProductController.getProduct);
```

Comment:
I suggest you separate product search route separate from the `getProduct` that had been defined because there will also be a need to get a single product using only for example when designing a single product page that contains all its information. This page will be separate from the one containing results of a search query. I have provided examples below:
Search query results: https://www.backmarket.com/en-us/search?q=macbook%20pro
Single product page: https://www.backmarket.com/en-us/p/macbook-pro-133-inch-core-i5-8gb-ssd-128-gb/ba152fea-da77-408d-902b-0996a42ccc82#l=11
I hope this helps.